app/views/editorial/nodes/_editor_injection.haml

```diff
@@ -6,7 +6,22 @@
       var text = $('#node_content_body')[0].value,
         target = $('#editor-preview')[0],
         converter = new showdown.Converter({tables: true}),
-        html = converter.makeHtml(text);
+        // default filter settings from https://github.com/punkave/sanitize-html made explicit below
+        html = converter.makeHtml(sanitizeHtml(text), {
+        allowedTags: [ 'h3', 'h4', 'h5', 'h6', 'blockquote', 'p', 'a', 'ul', 'ol',
```

Comment:
The app will render `h1`, `h2`, and `img` tags if they're used in published content, so I think we should include in `allowedTags` so the live preview is accurate. Unless having `img` leaves the security flaw we're trying to fix?